compiler/src/org.graalvm.compiler.truffle.compiler/src/org/graalvm/compiler/truffle/compiler/phases/inlining/CallNode.java

```diff
@@ -358,6 +362,10 @@ public boolean isTrivial() {
         return trivial;
     }
 
+    public boolean isExcluded(String optionsExcludeInlining) {
+        return !optionsExcludeInlining.isEmpty() && getTruffleAST().getName().contains(optionsExcludeInlining);
```

Comment:
The node should not be aware of the notion of exclusion, it's up to the policy (the CallNode already has a `getName` that exposes the `truffleAST.getName()`

Reply:
Fixed! I removed this function from CallNode and did the check directly within the policy - https://github.com/oracle/graal/pull/3524/files#diff-e7b7dbb1dab8bdbd05febd1974d7ee8c960438950b1f69932ea14df31bf0bc4bR140